content/blog/2022-07-08-announcing-poetry-1-1-14.md

```diff
@@ -0,0 +1,24 @@
+---
+layout: single
+title: "Announcing Poetry 1.1.14"
+date: 2022-07-08 00:00:00
+categories: [releases]
+tags: ["1.x", "1.1"]
+---
+
+The Poetry team is pleased to announce the immediate availability of Poetry **1.1.14**.
+
+<!--more-->
+
+This release is a bugfix release.
+
+If you have a previous version of Poetry installed via the [official installer](/docs/#installation),
+getting Poetry **1.1.14** is as easy as:
+
+```bash
+$ poetry self update
+```
+
+## Fixed
+
+- Fixed an issue where dependencies hashes could not be retrieved when locking due to a breaking change on PyPI JSON API ([#5973](https://github.com/python-poetry/poetry/pull/5973))
```

Comment:
"dependencies hashes" sounds weird to me (but I can be wrong). Maybe, "dependency hashes" or "dependencies' hashes" or "hashes of dependencies"? If you decide to change it same for `history.md`.

Reply:
Just a fly-by suggested rewording:
```
- Fixed an issue where retrieval of dependencies hashes during locking could fail due to a breaking change on the PyPI JSON API ([#5973](https://github.com/python-poetry/poetry/pull/5973))
```

Reply:
Oh yeah sorry, not a native English speaker, so I wasn't sure about the phrasing.

As mentioned in https://github.com/python-poetry/poetry/pull/5973#issuecomment-1179199250, we could add a note that users may need to clear `pypi` cache to fully solve the issue.

Reply:
I just copied the wording from the Changelog. That's why I would prefer keeping it.

Added a note about clearing the cache now.